README.md

```diff
@@ -2,30 +2,49 @@
 
 [![Build Status](https://api.travis-ci.org/sbt/sbt-conductr.png?branch=master)](https://travis-ci.org/sbt/sbt-conductr)
 
-sbt-conductr is an sbt plugin designed to facilitate the development lifecycle at the stage where deployment
-to Typesafe ConductR is required.
+sbt-conductr is an sbt plugin provides commands that communicate with ConductR from within an sbt session. It can
+also be used in conjunction with [sbt-bundle](https://github.com/sbt/sbt-bundle#conductr-bundle-plugin)
+ to deploy your application or service to ConductR without leaving the comforts of sbt.
 
 ## Usage
 
-With the ConductR running you can upload a file using sbt:
+sbt-conductr is enabled by simply declaring its dependency:
 
-```bash
-cd sbt-conductr-tester/
-sbt
+```scala
+addSbtPlugin("com.typesafe.conductr" % "sbt-conductr" % "1.0.0")
 ```
 
-The above puts you into the console of a test project. The test project is configured to use the sbt-native-packager
-to produce RR bundles, and declares some scheduling configuration. To learn more check out the build.sbt file of this
-same directory.
+The plugin has no requirements for other plugins to be enabled. 
+
+Unless the ConductR is running at `127.0.0.1:9005`, and instead supposing that it is running at
+`192.168.59.103:9005` then you will typically issue the following command from the sbt console:
+
+```
+controlServer 192.168.59.103:9005
+```
 
-Produce a bundle by typing:
+This then sets the sbt session up to subsequently communicate with the ConductR at 192.168.59.103 on port 9005. 
+You may type commands such as:
+
+```
+conduct info
+```
+
+...which will obtain information on the ConductR cluster.
+
+If you have [sbt-bundle](https://github.com/sbt/sbt-bundle#conductr-bundle-plugin) enabled e.g.:
```

Comment:
`sbt-bundle` is automatically enabled if `sbt-conductr` is used?! Can we change the README.md accordingly.